test/binCases/watch/info-verbosity-off/info-verbosity-off.test.js

```diff
@@ -1,13 +1,41 @@
 "use strict";
 
-jest.setTimeout(10E6);
+jest.setTimeout(10e6);
 /* eslint-disable node/no-unsupported-features  */
 /* eslint-disable node/no-unsupported-features/es-syntax  */
 
-const { runWatch } = require("../../../testUtils");
+const { extractSummary, extractHash, appendDataIfFileExists, runAndGetWatchProc } = require("../../../testUtils");
+const fs = require("fs");
```

Comment:
native requires before local ones

Reply:
Can you be a little bit more informative? Is this is a styling guide that we use? or is there a downside using it like this?

Reply:
```
const fs = require('fs')
const {extractSummary} = ...
```

No, but it's standard convention in node